src/cryptonote_core/cryptonote_core.cpp

```diff
@@ -1961,12 +1941,10 @@ namespace cryptonote
   //-----------------------------------------------------------------------------------------------
   void core::pause_mine()
   {
-    m_miner.pause();
   }
   //-----------------------------------------------------------------------------------------------
   void core::resume_mine()
   {
-    m_miner.resume();
   }
   //-----------------------------------------------------------------------------------------------
```

Comment:
These functions now do nothing and can be removed, but I don't see these calls being handled anymore? 

Places where core expects to be able to pause/resume the miner to avoid race conditions are unhandled. Preserving that behaviour or verifying that the miner can continue running where we used to expect it to pause/resume will need some investigation or some more architectural work to get it to behave nicely.